lib/kvdb/kvdb_ctxn.c

```diff
@@ -645,6 +674,8 @@ kvdb_ctxn_reset(struct kvdb_ctxn *handle)
     ctxn->ctxn_seqref = HSE_SQNREF_INVALID;
     ctxn->ctxn_expired = false;
 }
+// It is hard to believe that a txn reset only needs to initialize 2
+// members out of the large complicated ctxn struct?
```

Comment:
That's only called when getting a ctxn from the cache, but to get on the cache the ctxn must have been aborted, which cleans up the state.  Now I have a headache..